### PR TITLE
MessageBox replaced with Alert in login dialog

### LINKED ISSUE #322: Login dialog and error message box overlapping
#### My reason:

screen shot 2017-09-12 at 11 04 25 pm

####  App Version:	
All

####  Config file:	
full config file provided in conf directory.

#### Additional information:

screenshot attached.
 
 - Container:
   - I use local environment

#### Additional verbose log:
```sh
warn --- config file  - /Users/ayush/.config/verdaccio/config.yaml
 warn --- http address - http://localhost:4873/ - verdaccio/2.3.6
 http <-- 200, user: undefined(127.0.0.1), req: 'GET /', bytes: 0/1284
 http <-- 200, user: undefined(127.0.0.1), req: 'GET /-/static/style.8410f14de467c42d2f6c14c7df197f3f.css', bytes: 0/47471
 http <-- 200, user: undefined(127.0.0.1), req: 'GET /-/static/main.29190c77c9249937b209.js', bytes: 0/441993
 http <-- 200, user: undefined(127.0.0.1), req: 'GET /-/static/logo.png', bytes: 0/25736
 http <-- 200, user: undefined(127.0.0.1), req: 'GET /-/verdaccio/packages', bytes: 0/1103
 http <-- 401, user: undefined(127.0.0.1), req: 'POST /-/verdaccio/login', error: bad username/password, access denied
 http <-- 401, user: undefined(127.0.0.1), req: 'POST /-/verdaccio/login', error: bad username/password, access denied
 http <-- 401, user: undefined(127.0.0.1), req: 'POST /-/verdaccio/login', error: bad username/password, access denied
```

### PR DESCRIPTION
**Pull request by ayusharma**
**Type:** bug 

The following has been addressed in the PR:

*  There is a related issue #322 
*  Unit or Functional tests are included in the PR

**Description:**

I have replaced MessageBox with Alert.

Resolves #322

### CODE REVIEW

File: src/webui/src/components/Header/index.js [hunk 1]
```diff
@@ -54,11 +60,16 @@ export default class Header extends React.Component {
       storage.setItem('username', resp.data.username);
       location.reload();
     } catch (e) {
+      let errorObj = {
```
Comment (juanpicado) [inline src/webui/src/components/Header/index.js]: It could be `const`

File: src/webui/src/components/Header/index.js [hunk 2]
```diff
@@ -137,6 +148,13 @@ export default class Header extends React.Component {
           onCancel={ () => this.toggleLoginModal() }
         >
           <Dialog.Body>
+            { this.state && this.state.loginError &&
```
Comment (juanpicado) [inline src/webui/src/components/Header/index.js]: I think you don't have to check for `this.state `

File: src/webui/src/components/Header/index.js (hunk 1 above)
Comment (ayusharma) [reply src/webui/src/components/Header/index.js]: Yes. I have updated PR.

**Comment by juanpicado** [comment]
@ayusharma conflicts

**Comment by ayusharma** [comment]
Rebased it again. Resolved conflict.

**Comment by juanpicado** [comment]
Good feature ! Thanks @ayusharma

**Pull request merged**